README.md

```diff
@@ -16,21 +16,27 @@ All contributors are welcome! If you would like to contribute:
 
 ### Setup
 1. Fork this repo and clone it locally by running `git clone https://github.com/<your_username>/rideboard.git`
-2. `cd rides/`
+2. `cd csh-rideboard/`
 2. Create a python virtual environment, activate it and install requirements.
   - `virtualenv rides-venv`
   - `source rides-venv/bin/activate`
   - `pip install -r requirements.txt`
-5. You will need the _LDAP DN_, _LDAP PW_, _MySQL Databse_, and _OIDC Secret_. You can create your own if you are hosting your own application or ask me for these if you would like to contribute to CSH.
+5. You will need many credentials to run, see the exports below. You can create your own if you are hosting your own application or ask an rtp for these if you would like to contribute to CSH. I'd recommend making a config.sh to automatically do the exports you need. It is in the .gitignore so it will not go to the remote repository.
```

Comment:
How are you using this `config.sh` file? Are you just doing `source config.sh`? I think it's more typical to name this file `.env`.

Reply:
config.sh basically just automated the exports you'd have to do to test it every time. Similar to config.py if you did that method

Reply:
`config.sh` is what this file's been called in this repo for a long time, Ayush added it to the gitignore 3 years ago.